### PR TITLE
sync: fix getting URL from contract in compatibility mode, and set sequential by default

### LINKED ISSUE #2806: Synchronized must use by default the sequential mode
# Rationale

The parallel mode is still pending to be verified, for that reason the default must be the sequential mode

# Implementation

Change config

### PR DESCRIPTION
**Pull request by joanestebanr**
Closes #2806

### What does this PR do?
- If are not able to get trustedURL from contract ask to oldContract
- Set sync from L1 mode to `sequential` by default

### Reviewers

Main reviewers:

-- @ARR552 

Codeowner reviewers:

- @-Alice
- @-Bob

### CODE REVIEW

File: etherman/etherman.go [hunk 1]
```diff
@@ -1665,7 +1665,13 @@ func (etherMan *Client) ApprovePol(ctx context.Context, account common.Address,
 
 // GetTrustedSequencerURL Gets the trusted sequencer url from rollup smc
 func (etherMan *Client) GetTrustedSequencerURL() (string, error) {
-	return etherMan.ZkEVM.TrustedSequencerURL(&bind.CallOpts{Pending: false})
+	url, err := etherMan.ZkEVM.TrustedSequencerURL(&bind.CallOpts{Pending: false})
+	//TODO: remove this code because is for compatibility with oldZkEVM
+	if err != nil || (len(url) == 0) {
+		// Getting from oldZkEVM Contract
```
Comment (ARR552) [inline etherman/etherman.go:1671]: I think it would be nice to add a log here (debug level)

File: etherman/etherman.go [hunk 2]
```diff
@@ -1665,7 +1665,13 @@ func (etherMan *Client) ApprovePol(ctx context.Context, account common.Address,
...
+		return etherMan.OldZkEVM.TrustedSequencerURL(&bind.CallOpts{Pending: false})
+	}
+	return url, err
```
Comment (ARR552) [inline etherman/etherman.go]: this err will always be nil

File: etherman/etherman.go [hunk 3]
```diff
@@ -1665,7 +1665,13 @@ func (etherMan *Client) ApprovePol(ctx context.Context, account common.Address,
 
 // GetTrustedSequencerURL Gets the trusted sequencer url from rollup smc
 func (etherMan *Client) GetTrustedSequencerURL() (string, error) {
-	return etherMan.ZkEVM.TrustedSequencerURL(&bind.CallOpts{Pending: false})
+	url, err := etherMan.ZkEVM.TrustedSequencerURL(&bind.CallOpts{Pending: false})
+	//TODO: remove this code because is for compatibility with oldZkEVM
+	if err != nil || (len(url) == 0) {
```
Comment (ARR552) [inline etherman/etherman.go]:
```suggestion
	if err != nil || url == "" {
```

**Review by ARR552 — CHANGES_REQUESTED** [review]

File: etherman/etherman.go (hunk 2 above)
Comment (joanestebanr) [reply etherman/etherman.go]: what do you suggest: `return url, nil`?

**Review by ARR552 — APPROVED** [review]

**Pull request merged**